.github/workflows/github-pages.yml

```diff
@@ -13,27 +13,25 @@ jobs:
       with:
         submodules: true
         persist-credentials: false
-    
+
     - name: Info
       run: echo "Running on branch $(basename ${{ github.ref }})"
-    
+
     - name: Install dependencies
       run:  npm install --production=false
-    
+
     - name: Create build dir
-      run: mkdir docs_out    
-    
+      run: mkdir docs_out
+
     - name: Build documentation # only if this was triggered from latest
       if: github.ref == 'refs/heads/latest' 
-      # TODO: replace these with `npm docs` and `npm postdocs` once you are sure everything works
       run: |
-        npx typedoc --out docs_out src
-        npx copyfiles images/* docs_out
-        npx copyfiles .nojekyll docs_out
-    
+        npm docs
+        npm postdocs
```

Comment:
have you tested this? `npm docs` is not the same as `npm run docs`, the former will try to discern the location of the project's documentation and the latter will run a custom script. i'm not sure how/if github handles this distinction

Reply:
good point. I *think* I did, but this is a month old, I don't really remember. I'll try it locally first and see what happens

Reply:
you are right, I'll fix this